.github/workflows/test_runner.yml

```diff
@@ -206,3 +218,15 @@ jobs:
     with:
       safe-checkout: ${{ needs.check-tag.outputs.checkout-sha }}
     secrets: inherit
+
+  # This test depends on all blocking tests and indicates whether they all suceeded.
+  all_blocking_tests:
+    name: All Blocking Tests
+    needs: [bazel, cpp, java, python, ruby, php, php-ext, csharp, objectivec, rust, upb, staleness]
```

Comment:
We should probably add IFTTT comments even if it's unenforceable from GH.  I can't think of any other way to notify people when this list goes stale

Reply:
Yea that's a good point.  I don't have any great ideas..  Maybe we could parse this file in a google3 test to validate this condition?  There must be an internal yaml parser somewhere..

Reply:
Since our tests are already language specific, the only way that this goes stale is if someone adds a new language to test. I think instead we should just document how to start tests for a language and include updating this. 

I agree it's not ideal I just don't think it's worth the work to parse this for the very few times I think we will get an addition here.